src/startupscriptgenerator/src/common/environmentVariablesHelper.go

```diff
@@ -23,3 +23,11 @@ func GetBooleanEnvironmentVariable(key string) bool {
 	}
 	return result
 }
+
+func GetEnvironmentVariable(key string) string {
+	value, ok := os.LookupEnv(key)
+	if !ok {
+		panic(fmt.Sprintf("'%s' not set\n", key))
```

Comment:
Setting the `PRE_RUN_COMMAND` is optional, so we should not fail running the app if it is not present.